packages/rspack/src/Compilation.ts

```diff
@@ -130,6 +132,8 @@ export class Compilation {
 	outputOptions: OutputNormalized;
 	compiler: Compiler;
 	resolverFactory: ResolverFactory;
+	// TODO: maybe we can replace `resolverFactory` to `nativeResolverFactory`
+	nativeResolverFactory: NativeResolverFactory;
```

Comment:
Do we still need to keep resolverFactory?

Reply:
I'm not sure, but I intend to keep it.